features/rni.feature

```diff
@@ -199,7 +199,63 @@ Functionaliteit: RNI-deelnemer voor geleverde gegevens
+      | rni-deelnemer (88.10)        | 0101                                        |
+      | omschrijving verdrag (88.20) | Belastingverdrag tussen België en Nederland |
+      En de persoon heeft de volgende 'inschrijving' gegevens
+      | naam                         | waarde                                      |
+      | indicatie geheim (70.10)     | 0                                           |
+      | rni-deelnemer (88.10)        | 0101                                        |
+      | omschrijving verdrag (88.20) | Belastingverdrag tussen België en Nederland |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                         | waarde                               |
+      | land (13.10)                 | 5010                                 |
+      | rni-deelnemer (88.10)        | 0201                                 |
+      | omschrijving verdrag (88.20) | Artikel 45 EU-Werkingsverdrag (VWEU) |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                    |
+      | type                | RaadpleegMetBurgerservicenummer           |
+      | burgerservicenummer | 000000085                                 |
+      | fields              | burgerservicenummer,rni                   |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000085 |
+      En heeft de persoon een 'rni' met de volgende gegevens
+      | naam                   | waarde                                          |
+      | deelnemer.code         | 0101                                            |
       | deelnemer.omschrijving | Belastingdienst (inzake heffingen en toeslagen) |
       | omschrijvingVerdrag    | Belastingverdrag tussen België en Nederland     |
       | categorie              | Inschrijving                                    |
+      En heeft de persoon een 'rni' met de volgende gegevens
+      | naam                   | waarde                                          |
+      | deelnemer.code         | 0101                                            |
+      | deelnemer.omschrijving | Belastingdienst (inzake heffingen en toeslagen) |
+      | omschrijvingVerdrag    | Belastingverdrag tussen België en Nederland     |
+      | categorie              | Persoon                                         |
+      En heeft de persoon een 'rni' met de volgende gegevens
+      | naam                   | waarde                                          |
+      | deelnemer.code         | 0101                                            |
+      | deelnemer.omschrijving | Belastingdienst (inzake heffingen en toeslagen) |
+      | omschrijvingVerdrag    | Belastingverdrag tussen België en Nederland     |
+      | categorie              | Verblijfplaats                                  |
+      En heeft de persoon een 'rni' met de volgende gegevens
```

Comment:
Waarom wordt er rni geleverd voor nationaliteit en overlijden? Ik zie geen gegeven stap voor deze twee categorieën

Reply:
Ik vermoed dat dat een fout is in de GBA-API, maar dat weet ik niet zeker. Ik kan het anders niet anders verklaren.